.github/workflows/frodo-kem.yml

```diff
@@ -0,0 +1,134 @@
+name: frodo-kem
+
+on:
+  pull_request:
+    paths:
+      - "frodo-kem/**"
+      - "Cargo.*"
+    paths-ignore: 
+      - README.md
+      - SECURITY.md
+      - LICENSE-Apache
+      - LICENSE-MIT
+  push:
+    branches: master
+      - "frodo-kem/**"
+      - "Cargo.*"
+    paths-ignore:
+      - README.md
+      - SECURITY.md
+      - LICENSE-Apache
+      - LICENSE-MIT
+
+env:
+  CARGO_INCREMENTAL: 0
+  CARGO_TERM_COLOR: always
+  RUST_LOG_STYLE: always
+  RUSTFLAGS: "-Dwarnings"
+  RUSTDOCFLAGS: "-Dwarnings"
```

Comment:
We typically use attributes in lib.rs for this, i.e. `#![deny(warnings)]`

Reply:
No, we [use](https://github.com/RustCrypto/hashes/blob/master/.github/workflows/sha2.yml#L16-L18)  `RUSTFLAGS: "-Dwarnings"` to fail CI job on compiler warnings. I can not recall a single crate with `#![deny(warnings)]`.

Although, we usually keep rustdoc, rustfmt, and clippy jobs in a separate workflow.

Reply:
Oh whoops, my bad, not sure why I said that